examples/with-docker/Dockerfile

```diff
@@ -45,17 +47,17 @@ ENV NODE_ENV production
 RUN addgroup --system --gid 1001 nodejs
 RUN adduser --system --uid 1001 nextjs
 
-COPY --from=builder /app/public ./public
-
-# Set the correct permission for prerender cache
-RUN mkdir .next
-RUN chown nextjs:nodejs .next
-
 # Automatically leverage output traces to reduce image size
 # https://nextjs.org/docs/advanced-features/output-file-tracing
+COPY --from=builder --chown=nextjs:nodejs /app/public ./public
 COPY --from=builder --chown=nextjs:nodejs /app/.next/standalone ./
 COPY --from=builder --chown=nextjs:nodejs /app/.next/static ./.next/static
 
+# Set the correct permission for prerender cache
+RUN mkdir -p .next/cache
+RUN chown -R nextjs:nodejs .next
+RUN chmod -R 777 .next/cache
```

Comment:
777 in production is security threat and not safe

Reply:
you can view my site Dockerfile, I have tested it with no permission issue (or not yet, especially .next/cache issue)

https://github.com/dodyagung/dodyagung.com/blob/dodyagung.com/Dockerfile

cc @Kinbaum @samcx